commands/utils/packagehandlers/packagehandlers_test.go

```diff
@@ -542,9 +579,14 @@ func getTestDataDir(t *testing.T, directDependency bool) string {
 	return testdataDir
 }
 
-func createTempDirAndChDir(t *testing.T, testdataDir string, tech coreutils.Technology) func() {
+func createTempDirAndChDir(t *testing.T, testdataDir string, tech coreutils.Technology, extraArgs ...string) func() {
```

Comment:
```suggestion
func createTempDirAndChdir(t *testing.T, testdataDir string, tech coreutils.Technology) func() {
```

Reply:
I need extra args so specify the path to yarn testcases (since I split between versions)